src/web/app/pom.xml

```diff
@@ -1073,6 +1073,14 @@
           <artifactId>gs-geofence-server</artifactId>
           <version>${project.version}</version>
         </dependency>
+        <!-- Older versions of h2 fail in drop/create tables   -->
```

Comment:
Is it needed? The dependency towards gs-geofence-server should be enough, it's already declared as a dependency there.

Reply:
Without it, mvn will usee the 1.1 version :|